### DISCUSSION TITLE
Is there any way to find or navigate to bad instructions?

### DISCUSSION THREAD

**Discussion by hippietrail** [NationalSecurityAgency/ghidra#7140] [Q&A] [answered]
In my efforts to understand how different systems use Motorola 68000 A-line and F-line traps I'm looking at lots of different formats but it's very hard to find bad instructions. When you do hit one it looks like this: 
<img width="1221" alt="image" src="https://github.com/user-attachments/assets/84aea97e-e198-49f9-ba74-f93e1b75f8c9">

There are the little red X's in the left column but there is no indication in the right columns with colour-coded info on other things to help us navigate.

I can't find anything in the online help or in the menus or in Search / Program Text. Searching for `?` matches every character. Searching for `?` in Instruction Mnemonics matches nothing.

Does support for this exist and I just can't find it? Or are there no features for this other than the red X's that you can only see if you'e in the part of the listing with the bad instruction?

If there are no such features I will go ahead and propose them after hearing feedback here.

**Comment by ryanmkurtz** [comment, marked as answer]
Don't those little red X's make entries in the Bookmark window?

**Reply by hippietrail** [reply]
Yes they do! One of the windows I had never used but the popup tooltip should've given me enough of a clue!

**Comment by hippietrail** [comment]
It turns out bad instructions are shown in the "overview bar" on the right. But bad addresses show up exactly the same way and are far far more common, so I couldn't see the trees for the forest.
